BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/BottomNavigation/SkontoNavigationBarBottomAdapter.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SkontoNavigationBarBottomAdapter.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import GiniCaptureSDK
+import UIKit
+
+/**
+Protocol for injecting a custom bottom navigation bar on Skonto.
+
+- note: Bottom navigation only.
+*/
+public protocol SkontoNavigationBarBottomAdapter: InjectedViewAdapter {
+    /**
+     *  Set the callback for the proceed button action.
+     *
+     * - Parameter callback: An  action callback, which should be retained and called in proceed button action method
+     */
+    func setProceedButtonClickedActionCallback(_ callback: @escaping  () -> Void)
+
+    /**
+     *  Set the callback for the help button action.
+     *
+     * - Parameter callback: An  action callback, which should be retained and called in help button action method
+     */
+    func setHelpButtonClickedActionCallback(_ callback: @escaping  () -> Void)
+
+    /**
+     *  Set the callback for the back button action.
+     *
+     * - Parameter callback: An  action callback, which should be retained and called in back button action method
+     */
+    func setBackButtonClickedActionCallback(_ callback: @escaping  () -> Void)
+
+    /**
+     *  Set the total price on the bottom navigation bar. Called when the total price changes
+     *
+     * - Parameter price: A string which contains the currency and the price
+     */
+    func updateTotalPrice(priceWithCurrencySymbol price: String?)
+
+    /**
+     *  Set the proceed button state. Called when state of the button should be changed
+     *
+     * - Parameter enabled: A bool value to reflect the state of the button
+     */
+    func updateProceedButtonState(enabled: Bool)
+
+    /**
+     *  Set the discount value on the bottom navigation bar. Called when Skonto applies
+     *
+     * - Parameter discount: A string which contains the value of discount
+     */
+    func updateDiscountValue(with discount: String?)
+
+    /**
+     *  Set the discount badge state. Called when Skonto applies
+     *
+     * - Parameter enabled: A bool value to reflect the state of the badge
+     */
+    func updateDiscountBadge(enabled: Bool)
```

Comment:
[clarification] I'm not sure we need this here. Let's clean them after finishing the backend integration. Please add a TODO.

Reply:
@ValentinaIancu-Gini it's used to show badge or not, and updateDiscountValue used to insert text in it. Without updateDiscountBadge, updateDiscountValue wouldn't display anything
If we need some another business logic or we still need TODO, let me know please

Reply:
I still don't know if we really need this to be exposed to external use or not. Since in the RA there is nothing exposed related to recalculation of the total price. Let's talk a bit about this.